plugins/lookup/revbitspss.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, RevBits <info@revbits.com>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: revbitspss
+author: RevBits (@RevBits) <info@revbits.com>
+short_description: Get secrets from RevBits PAM server
```

Comment:
Lowercase is preferred in short description
```suggestion
short_description: get secrets from RevBits PAM server
```

Reply:
That's not true. We prefer uppercase. (If you scroll through https://docs.ansible.com/ansible/latest/collections/community/general/, you will see that the large majority uses upper-case.)

Reply:
No, I'm positive!
Unless I'm not.
Will search PR history.
Yeah, he is right. :facepalm: 

I recall there was one item somewhere that everybody went for uppercase but it was supposed to be lowercase.